HTML/Excersize2/script.js

```diff
@@ -0,0 +1,22 @@
+let clock = document.getElementById("clock");
+let daytime = document.getElementById("daytime");
+
+
+function getTime(){
+    let date = new Date();
+    let hour = date.getHours();
+    let min = date.getMinutes();
+    let sec = date.getSeconds();
```

Comment:
similarly, here as well, should have used const.

Reply:
yes done